tests/testthat/test-sun-methods.R

```diff
@@ -97,15 +98,15 @@ test_that("solarnoon() can calculate the solar noon", {
         )
       ),
       class = "data.frame",
-      row.names = c(NA, -1L)
+      row.names = 'newlon' 
```

Comment:
Investigate unexpected behavior in tests - passing locally but not in github actions.  This stems from a mismatch in rownames.  Locally the expected output had to be changed to:
```
    structure(
      list(
        day_frac = 0.276199993629954,
        time = structure(1103863063.67945,
          tzone = "EET",
          class = c("POSIXct", "POSIXt")
        )
      ),
      class = "data.frame",
      row.names = 'newlon' 
    )
```
    
Originally:

row.names = c(NA, -1L)